x/authz/spec/01_concepts.md

```diff
@@ -0,0 +1,37 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Authorization
+Any concrete type of authorization defined in the `x/authz` module must fulfill the `Authorization` interface outlined below. Authorizations determine exactly what privileges are granted. They are extensible and can be defined for any Msg service method even outside of the module where the Msg method is defined. Authorizations use the new `ServiceMsg` type from [ADR 031](../../../docs/architecture/adr-031-msg-service.md).
```

Comment:
```suggestion
Any concrete type of authorization defined in the `x/authz` module must fulfill the `Authorization` interface outlined below. Authorizations determine exactly what privileges are granted. They are extensible and can be defined for any Msg service method even outside of the module where the Msg method is defined. Authorizations use the new `ServiceMsg` type from [ADR 031](../../../architecture/adr-031-msg-service.md).
```
The ADR link was not workin when I rendered docs locally, this should fix it.

Reply:
Unfortunately, we need to choose between having these docs files render proper links on **either** bowsing .md files on github OR when viewing on docs.cosmos.network. The two are incompatible when linking btw module & ADR docs as all the modules get copied into the `./docs` folder for building docs.cosmos.network. My preference is to have the links render correctly when viewed on docs.cosmos.network

Reply:
The other option (if we don't want to do relative path), would be to have them link to absolute paths as rendered on docs.cosmos.network.

This could be done with `[ADR 031](/architecture/adr-031-msg-service.md)` which would always link to the default version of the docs (most recent released version), or we could hardcode a specific version and/or master like so:
`[ADR 031](/master/architecture/adr-031-msg-service.md)`.

My preference is for the relative path (as in my suggested change above).